### PR TITLE
Fix the max-height bug for the expanded Document editor view

### PR DESCRIPTION
**Pull request by JedWatson**
As mentioned by @gautamsi in #4883

This feels like a bit of a hack, but it fixes the bug and doesn't seem to have any unintended side effects.

@jossmac wouldn't mind your eyes on this, in case there's something I've missed

### CODE REVIEW

**Comment by jossmac** [comment]
> wouldn't mind your eyes on this, in case there's something I've missed

@JedWatson Should be fine, though I'm not super familiar with the code anymore. The "maximised mode" was supposed to fill below the header, is that still the case?

**Review by jossmac — APPROVED** [review]

**Comment by gautamsi** [comment]
this does not work as 100% height hides content at bottom by same height of offset from top.

File: packages-next/admin-ui/src/pages/ItemPage/index.tsx [hunk 1]
```diff
@@ -577,7 +577,7 @@ const ColumnLayout = (props: HTMLAttributes<HTMLDivElement>) => {
   const { spacing } = useTheme();
 
   return (
-    <Container css={{ position: 'relative' }}>
+    <Container css={{ position: 'relative', height: '100%' }}>
```
Comment (gautamsi) [inline packages-next/admin-ui/src/pages/ItemPage/index.tsx:580]: should this be leaving 
```suggestion
    <Container css={{ position: 'relative', height: calc('100%' - 20px) }}>
```
I am not sure how exactly this should be done.

Comment (jossmac) [reply packages-next/admin-ui/src/pages/ItemPage/index.tsx:580]: Hey, I have the context for this now. Applying `height: 100%` is a fine approach. I'd just add a comment why it's there

Comment (jossmac) [inline packages-next/admin-ui/src/pages/ItemPage/index.tsx:580]:
```suggestion
      // this container must be relative to catch absolute children
      // particularly the "expanded" document-field, which needs a height of 100%
      <Container css={{ position: 'relative', height: '100%' }}>
```

Comment (JedWatson) [reply packages-next/admin-ui/src/pages/ItemPage/index.tsx:582]: @gautamsi we don't need to exclude any height there because the container sorts it out

**Pull request merged**